minvws/nl-kat-coordination
# Fix report names for scheduled reports

### Changes
This PR fixes the following bugs:

- oois_count counts the subreports, but should count the amount of selected objects
- "{ooi}" doesn't seem to work anymore
- Object in name of subreports should be human readable 

### Issue link
Closes #3709

### Demo

_Please add some proof in the form of screenshots or screen recordings to show (off) new functionality, if there are interesting new features for end-users._

### QA notes

1. Create a normal report with one object and one report type
2. Schedule it
3. Change the schedule to every minute (* * * * *)
4. Check History page to see the generated report after a minute
5. You shouldn't see"{ooi}" in the title

----------

1. Create a normal report with multiple objects and multiple report types
2. Schedule it
3. Change the schedule to every minute (* * * * *)
4. Check History page to see the generated report after a minute
5. Open the table row to see the subreports
7. You should see that that the names are something like "Open Ports Report for mispo.es" (instead of Hostname|internet}mispo.es")

### Not fixed
This PR doesn't fix the bug where the Aggregate Report always ends up with the name "Concatenated Report".
This bug is more complex than I thought it would be, that's why I've created a new bug report for it.

Ticket: #3725 

---

### Code Checklist






- [x] Tickets have been created for newly discovered issues. #3725 

---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
